.circleci/config.yml

```diff
@@ -75,6 +54,14 @@ jobs:
               bash build_tools/circle/push_doc.sh doc/_build/html/stable
             fi
 
+  # This noop job is required for the pipeline to exist, so that the
+  # documentation related jobs can be triggered.
+  noop:
+    docker:
+      - image: cimg/python:3.8.12
+    steps:
+      - run: echo "noop job"
```

Comment:
A small nitpick: having this here make the logs more explicit IMO (for me it was hard to understand why we needed to synchronisation between GitHub Actions and Circle CI. 
```suggestion
    - run: echo "This is no-op job for the pipeline to exist, so that it triggers Circle CI jobs pushing the artifacts of the documentation built via GitHub actions."
```

What do you think?

Reply:
Yea that makes sense.